src/components/ArticleList.jsx

```diff
@@ -5,13 +5,13 @@ import ArticleCard from "../components/ArticleCard";
 import Ad from "./Ad";
 import mercedesImg from "../images/mercedesAd.jpg";
 import lagavulinImg from "../images/lagavulinAd.jpg";
-import '../css/article.css'
-
-
+import "../css/article.css";
+import { connect, useSelector } from "react-redux";
```

Comment:
You don't have to connect it if you use useSelector.

Reply:
done